big_scape/comparison/extend.py

```diff
@@ -44,25 +44,26 @@ def check(pair: RecordPair, min_len: int, biosynth_check: bool) -> bool:
         biosynth_check: Whether to check for biosynthetic genes within the comparable
             region
     """
-    if biosynth_check:
-        if ComparableRegion.cds_range_contains_biosynthetic(
-            pair.record_a,
-            pair.comparable_region.a_start,
-            pair.comparable_region.a_stop,
-            False,
-            False,
-        ):
-            return True
-
-        if ComparableRegion.cds_range_contains_biosynthetic(
-            pair.record_b,
-            pair.comparable_region.b_start,
-            pair.comparable_region.b_stop,
-            False,
-            pair.comparable_region.reverse,
-        ):
-            return True
+    if biosynthetic_check(pair):
+        return True
+
+    a_len = pair.comparable_region.a_stop - pair.comparable_region.a_start
+    b_len = pair.comparable_region.b_stop - pair.comparable_region.b_start
+
+    if a_len >= min_len and b_len >= min_len:
+        return True
+
+    return False
+
```

Comment:
recheck if this check function can be completely removed, does not seem to be used anymore

Reply:
removed it